proj4/src/main/java/org/osgeo/proj4j/PolarCoordinate.java

```diff
@@ -0,0 +1,20 @@
+package org.osgeo.proj4j.util;
+
+import java.io.Serializable;
+
+public final class PolarCoordinate implements Serializable {
```

Comment:
After further review we've decided to leave this separate, as Complex and PolarCoordinate have slightly different semantics.